ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -163,27 +168,31 @@ const tailThrustOutputStrings = {
   iceLeft: {
     en: 'Double Knockback (<== Start on Left)',
     de: 'Doppel-Rückstoß (<== Starte Links)',
+    fr: 'Double poussée (<= Démarrez à gauche)',
     ja: '2連続ノックバック (<== 左から開始)',
     cn: '两次击退 (<== 左边开始)',
     ko: '넉백 2번 (<== 왼쪽에서 시작)',
   },
   iceRight: {
     en: 'Double Knockback (Start on Right ==>)',
     de: 'Doppel-Rückstoß (Starte Rechts ==>)',
+    fr: 'Double poussée (Démarrez à droite =>',
```

Comment:
```suggestion
    fr: 'Double poussée (Démarrez à droite ==>',
```
I think you'll need the long arrow `==>` here to prevent TTS from interpreting this as a follow-up instruction.

Reply:
Oh ... I never used TTS, so I didn't know that.